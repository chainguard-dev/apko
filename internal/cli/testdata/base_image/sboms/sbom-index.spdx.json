{
  "SPDXID": "SPDXRef-DOCUMENT",
  "name": "sbom-sha256:153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694",
  "spdxVersion": "SPDX-2.3",
  "creationInfo": {
    "created": "1970-01-01T00:00:00Z",
    "creators": [
      "Tool: apko (0.13.2)",
      "Organization: Chainguard, Inc"
    ],
    "licenseListVersion": "3.16"
  },
  "dataLicense": "CC0-1.0",
  "documentNamespace": "https://spdx.org/spdxdocs/apko/",
  "documentDescribes": [
    "SPDXRef-Package-sha256-153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694"
  ],
  "packages": [
    {
      "SPDXID": "SPDXRef-Package-sha256-153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694",
      "name": "sha256:153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694",
      "versionInfo": "sha256:153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694",
      "filesAnalyzed": false,
      "description": "Multi-arch image index",
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Chainguard, Inc.",
      "sourceInfo": "Generated at image build time by apko",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3A153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694?mediaType=application%2Fvnd.oci.image.index.v1%2Bjson",
          "referenceType": "purl"
        }
      ]
    },
    {
      "SPDXID": "SPDXRef-Package-sha256-2ef91a9967f2e1759ea49a8c01cf6a45dd9f9af71fe09bcf2b86175bc4a71314",
      "name": "sha256:2ef91a9967f2e1759ea49a8c01cf6a45dd9f9af71fe09bcf2b86175bc4a71314",
      "versionInfo": "sha256:2ef91a9967f2e1759ea49a8c01cf6a45dd9f9af71fe09bcf2b86175bc4a71314",
      "filesAnalyzed": false,
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Chainguard, Inc.",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "2ef91a9967f2e1759ea49a8c01cf6a45dd9f9af71fe09bcf2b86175bc4a71314"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3A2ef91a9967f2e1759ea49a8c01cf6a45dd9f9af71fe09bcf2b86175bc4a71314?arch=amd64\u0026mediaType=application%2Fvnd.oci.image.manifest.v1%2Bjson\u0026os=linux",
          "referenceType": "purl"
        }
      ]
    },
    {
      "SPDXID": "SPDXRef-Package-sha256-5a99438a9ced8193f1d71209d0b558fdc0b184aee5cf258e5f7aa9a6ab0f0671",
      "name": "sha256:5a99438a9ced8193f1d71209d0b558fdc0b184aee5cf258e5f7aa9a6ab0f0671",
      "versionInfo": "sha256:5a99438a9ced8193f1d71209d0b558fdc0b184aee5cf258e5f7aa9a6ab0f0671",
      "filesAnalyzed": false,
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Chainguard, Inc.",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "5a99438a9ced8193f1d71209d0b558fdc0b184aee5cf258e5f7aa9a6ab0f0671"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3A5a99438a9ced8193f1d71209d0b558fdc0b184aee5cf258e5f7aa9a6ab0f0671?arch=arm64\u0026mediaType=application%2Fvnd.oci.image.manifest.v1%2Bjson\u0026os=linux",
          "referenceType": "purl"
        }
      ]
    }
  ],
  "relationships": [
    {
      "spdxElementId": "SPDXRef-Package-sha256-153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694",
      "relationshipType": "VARIANT_OF",
      "relatedSpdxElement": "SPDXRef-Package-sha256-2ef91a9967f2e1759ea49a8c01cf6a45dd9f9af71fe09bcf2b86175bc4a71314"
    },
    {
      "spdxElementId": "SPDXRef-Package-sha256-153e3d9292715569c0c1fcd7f00671e156427e9da285efca4002c580a8cf1694",
      "relationshipType": "VARIANT_OF",
      "relatedSpdxElement": "SPDXRef-Package-sha256-5a99438a9ced8193f1d71209d0b558fdc0b184aee5cf258e5f7aa9a6ab0f0671"
    }
  ]
}
