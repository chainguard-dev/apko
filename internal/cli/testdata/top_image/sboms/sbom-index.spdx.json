{
  "SPDXID": "SPDXRef-DOCUMENT",
  "name": "sbom-sha256:ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b",
  "spdxVersion": "SPDX-2.3",
  "creationInfo": {
    "created": "1970-01-01T00:00:00Z",
    "creators": [
      "Tool: apko (devel)",
      "Organization: Chainguard, Inc"
    ],
    "licenseListVersion": "3.16"
  },
  "dataLicense": "CC0-1.0",
  "documentNamespace": "https://spdx.org/spdxdocs/apko/",
  "documentDescribes": [
    "SPDXRef-Package-sha256-ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b"
  ],
  "packages": [
    {
      "SPDXID": "SPDXRef-Package-sha256-ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b",
      "name": "sha256:ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b",
      "versionInfo": "sha256:ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b",
      "filesAnalyzed": false,
      "description": "Multi-arch image index",
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Alpine Linux",
      "sourceInfo": "Generated at image build time by apko",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3Aae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b?mediaType=application%2Fvnd.oci.image.index.v1%2Bjson",
          "referenceType": "purl"
        }
      ]
    },
    {
      "SPDXID": "SPDXRef-Package-sha256-1f6a8feb0923498f610445de79658aa551053e55aa36b1978858309f6214d0da",
      "name": "sha256:1f6a8feb0923498f610445de79658aa551053e55aa36b1978858309f6214d0da",
      "versionInfo": "sha256:1f6a8feb0923498f610445de79658aa551053e55aa36b1978858309f6214d0da",
      "filesAnalyzed": false,
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Alpine Linux",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "1f6a8feb0923498f610445de79658aa551053e55aa36b1978858309f6214d0da"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3A1f6a8feb0923498f610445de79658aa551053e55aa36b1978858309f6214d0da?arch=amd64\u0026mediaType=application%2Fvnd.oci.image.manifest.v1%2Bjson\u0026os=linux",
          "referenceType": "purl"
        }
      ]
    },
    {
      "SPDXID": "SPDXRef-Package-sha256-2df7af41200a78560d35b826f6fb02c68f98005c3a3b471e6a5a75c9b5b78223",
      "name": "sha256:2df7af41200a78560d35b826f6fb02c68f98005c3a3b471e6a5a75c9b5b78223",
      "versionInfo": "sha256:2df7af41200a78560d35b826f6fb02c68f98005c3a3b471e6a5a75c9b5b78223",
      "filesAnalyzed": false,
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Alpine Linux",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "2df7af41200a78560d35b826f6fb02c68f98005c3a3b471e6a5a75c9b5b78223"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3A2df7af41200a78560d35b826f6fb02c68f98005c3a3b471e6a5a75c9b5b78223?arch=arm64\u0026mediaType=application%2Fvnd.oci.image.manifest.v1%2Bjson\u0026os=linux",
          "referenceType": "purl"
        }
      ]
    }
  ],
  "relationships": [
    {
      "spdxElementId": "SPDXRef-Package-sha256-ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b",
      "relationshipType": "VARIANT_OF",
      "relatedSpdxElement": "SPDXRef-Package-sha256-1f6a8feb0923498f610445de79658aa551053e55aa36b1978858309f6214d0da"
    },
    {
      "spdxElementId": "SPDXRef-Package-sha256-ae32475c4438e3489ac832eb22e72bd00f5c41d2e8ef839742bcc0fe80bfd51b",
      "relationshipType": "VARIANT_OF",
      "relatedSpdxElement": "SPDXRef-Package-sha256-2df7af41200a78560d35b826f6fb02c68f98005c3a3b471e6a5a75c9b5b78223"
    }
  ]
}
