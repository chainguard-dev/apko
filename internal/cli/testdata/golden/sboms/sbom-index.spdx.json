{
  "SPDXID": "SPDXRef-DOCUMENT",
  "name": "sbom-sha256:714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54",
  "spdxVersion": "SPDX-2.3",
  "creationInfo": {
    "created": "1970-01-01T00:00:00Z",
    "creators": [
      "Tool: apko (devel)",
      "Organization: Chainguard, Inc"
    ],
    "licenseListVersion": "3.16"
  },
  "dataLicense": "CC0-1.0",
  "documentNamespace": "https://spdx.org/spdxdocs/apko/",
  "documentDescribes": [
    "SPDXRef-Package-sha256-714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54"
  ],
  "packages": [
    {
      "SPDXID": "SPDXRef-Package-sha256-714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54",
      "name": "sha256:714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54",
      "versionInfo": "sha256:714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54",
      "filesAnalyzed": false,
      "description": "Multi-arch image index",
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Chainguard, Inc.",
      "sourceInfo": "Generated at image build time by apko",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3A714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54?mediaType=application%2Fvnd.oci.image.index.v1%2Bjson",
          "referenceType": "purl"
        }
      ]
    },
    {
      "SPDXID": "SPDXRef-Package-sha256-43e3f561cb0910cb25ea304331454f74315d2edbcf68e2af172f8efa5c95fe19",
      "name": "sha256:43e3f561cb0910cb25ea304331454f74315d2edbcf68e2af172f8efa5c95fe19",
      "versionInfo": "sha256:43e3f561cb0910cb25ea304331454f74315d2edbcf68e2af172f8efa5c95fe19",
      "filesAnalyzed": false,
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Chainguard, Inc.",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "43e3f561cb0910cb25ea304331454f74315d2edbcf68e2af172f8efa5c95fe19"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3A43e3f561cb0910cb25ea304331454f74315d2edbcf68e2af172f8efa5c95fe19?arch=amd64\u0026mediaType=application%2Fvnd.oci.image.manifest.v1%2Bjson\u0026os=linux",
          "referenceType": "purl"
        }
      ]
    },
    {
      "SPDXID": "SPDXRef-Package-sha256-dcc96a7dc51e9f68b3402c82332b61325ae8e6f8ddc49159e7bafc30453bfc17",
      "name": "sha256:dcc96a7dc51e9f68b3402c82332b61325ae8e6f8ddc49159e7bafc30453bfc17",
      "versionInfo": "sha256:dcc96a7dc51e9f68b3402c82332b61325ae8e6f8ddc49159e7bafc30453bfc17",
      "filesAnalyzed": false,
      "downloadLocation": "NOASSERTION",
      "supplier": "Organization: Chainguard, Inc.",
      "primaryPackagePurpose": "CONTAINER",
      "checksums": [
        {
          "algorithm": "SHA256",
          "checksumValue": "dcc96a7dc51e9f68b3402c82332b61325ae8e6f8ddc49159e7bafc30453bfc17"
        }
      ],
      "externalRefs": [
        {
          "referenceCategory": "PACKAGE-MANAGER",
          "referenceLocator": "pkg:oci/golden@sha256%3Adcc96a7dc51e9f68b3402c82332b61325ae8e6f8ddc49159e7bafc30453bfc17?arch=arm64\u0026mediaType=application%2Fvnd.oci.image.manifest.v1%2Bjson\u0026os=linux",
          "referenceType": "purl"
        }
      ]
    }
  ],
  "relationships": [
    {
      "spdxElementId": "SPDXRef-Package-sha256-714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54",
      "relationshipType": "VARIANT_OF",
      "relatedSpdxElement": "SPDXRef-Package-sha256-43e3f561cb0910cb25ea304331454f74315d2edbcf68e2af172f8efa5c95fe19"
    },
    {
      "spdxElementId": "SPDXRef-Package-sha256-714bd8ce97dfb0b40f6ff0a8b82c05b1249101614fd77190d610ae11280bdc54",
      "relationshipType": "VARIANT_OF",
      "relatedSpdxElement": "SPDXRef-Package-sha256-dcc96a7dc51e9f68b3402c82332b61325ae8e6f8ddc49159e7bafc30453bfc17"
    }
  ]
}
